projects/canopy/src/styles/variables.scss

```diff
@@ -6,6 +6,7 @@
 @import 'variables/components/card';
 @import 'variables/components/data-point';
 @import 'variables/components/details';
+@import 'variables/components/filter-button';
```

Comment:
Hopefully we can share these variable across to the checkbox one too which is cool 👍 😎